изменил функцию передачи данных

ODN13 opened the pull request:
заменил функцию передачи информации о препятствиях на функцию основанную на onRequest() и write()

ODN13 committed:
изменил функцию передачи данных

заменил функцию передачи информации о препятствиях на функцию основанную на onRequest() и write()

FlameFlameFlame reviewed (changes requested):
Привести в порядок функцию transfers, начать нумеровать массивы с нуля

ODN13 committed:
сделали отправку данных на edison

написали функцию считывания и передачи массива данных на edison
сделали тестовый вывод на экран

FlameFlameFlame reviewed (approved):
В следующей ревизии сделать опрос датчиков по запросу с мастера, не постоянным.
